core/src/sigverify.rs

```diff
@@ -18,6 +18,8 @@ type TxOffsets = (Vec<u32>, Vec<u32>, Vec<u32>, Vec<u32>, Vec<Vec<u32>>);
 
 // The serialized size of Message::num_required_signatures.
 const NUM_REQUIRED_SIGNATURES_SIZE: usize = 1;
+// The serialized size of Message::num_credit_only_accounts.
+const NUM_CREDIT_ONLY_ACCOUNTS_SIZE: usize = 2;
```

Comment:
num_credit_only_accounts takes 2 octets?

Reply:
Yes, as both signed and unsigned keys can be credit-only

Reply:
yeah,  I see.  Ugh  `NUM_CREDIT_ONLY_ACCOUNTS_SIZE` and `NUM_REQUIRED_SIGNATURES_SIZE` feel like a footgun as they aren't really attached to `sdk/`

Reply:
Any reason not to have as part of message.rs and import from there to sigverify?

Reply:
That sounds way better